language-design.md

```diff
@@ -17,6 +17,27 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
+"Next to English, all computer programming languages are small; as we write code, we must stop now and then
+to define some new term that we will need to use in more than one place" [Steele, 1998].
+- This quote really made me think about the speakers objective in giving this talk. I had never really comapered
+programming languages to the English language because in my mind they were so different from each other. Once the 
+speaker presented this idea, I really stopped to think about how small and less complex programming languages
+are compared to spoken languages that have existes for thousands of years. Even a programming language like Python, 
```

Comment:
Yeah, the comparison to a spoken language is quite distinct. It definitely helps that programming languages are so simple in comparison. If it took me many years to learn the basics of a programming language like it does for a spoken language, I probably wouldn't bother. The best languages are the ones that have precisely enough defined and no more.